Tasks/DeployAzureResourceGroup/AzureResourceManagerHelper.ps1

```diff
@@ -102,6 +121,16 @@ function Get-Resources
     }
 }
 
+function Get-FQDN
+{
+     param([string]$resourceGroupName,
+           [string]$resourceName)
+
+     $publicIP = Get-AzurePublicIpAddress -ResourceGroupName $resourceGroupName -Name $resourceName
```

Comment:
Should we not handle this as  : -ErrorAction silentlycontinue, if so, then we need to see we can do Properties.DnsSettings.Fqdn. only under check - $publicIP not null.